core/src/test/scala/org/apache/spark/ExecutorAllocationManagerSuite.scala

```diff
@@ -1603,7 +1603,7 @@ class ExecutorAllocationManagerSuite extends SparkFunSuite {
       .set(config.DYN_ALLOCATION_TESTING, true)
       // SPARK-22864: effectively disable the allocation schedule by setting the period to a
       // really long value.
-      .set(TEST_SCHEDULE_INTERVAL, 10000L)
+      .set(TEST_SCHEDULE_INTERVAL, 30000L)
```

Comment:
So, this is for the same reason with SPARK-22864 at line 1604?

Reply:
@tgravescs  It happens again at: https://github.com/apache/spark/pull/29418/checks?check_run_id=975304054

BTW, this change only prevents the automatic invocation of `schedule()` for the second time but the first time invocation of `schedule()` always happens because the `initialDelay` is 0?

https://github.com/apache/spark/blob/8f0fef18438aa8fb07f5ed885ffad1339992f102/core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala#L250

Reply:
I'm not seeing. a test report in that run?
Yes but it seemed like the failures we were seeing before we after an interation or two or the test starting, although it did vary. If we can see the logs from it we should be able to see timing and possibly some log messages to tell us if that is the problem.

Reply:
> I'm not seeing. a test report in that run?

You need to scroll to the end of ` Run tests: core, unsafe, kvstore, avro` section to see the failed tests.

> Yes but it seemed like the failures we were seeing before we after an interation or two or the test starting, although it did vary. 

Ok, not aware of that...

> If we can see the logs from it we should be able to see timing and possibly some log messages to tell us if that is the problem.

@tgravescs A feasible way to debug it is to open a PR in your own fork repository and adds some `println` in the source code.  

@HyukjinKwon I think we still can not see the complete logs in GithubActions like the `unit-test.log` in Jenkins. I mean, even we download the archive logs after checks finished.

Reply:
I think I can try to upload that specific log file as an artifact. Let me take a look in coming few days.

Reply:
BTW, the `unit-tests.log` file is generated by `log4j`:

https://github.com/apache/spark/blob/b33066f42bd474f5f80b14221f97d09a76e0b398/core/src/test/resources/log4j.properties#L23

Reply:
thanks for updating and uploading the logs, let me know if you see a build that has this test failure and I'll look at the log